cypress/e2e/safe-apps/browser_permissions.cy.js

```diff
@@ -18,13 +18,14 @@ describe('Browser permissions tests', () => {
     main.acceptCookies()
   })
 
-  it('Verify a permissions slide to the user is displayed', () => {
+  // @TODO: unknown apps don't have permissions
+  xit('Verify a permissions slide to the user is displayed', () => {
     safeapps.clickOnContinueBtn()
     safeapps.verifyCameraCheckBoxExists()
     safeapps.verifyMicrofoneCheckBoxExists()
   })
 
-  it('Verify the selection can be changed, accepted and stored', () => {
+  xit('Verify the selection can be changed, accepted and stored', () => {
```

Comment:
@safe-global/safe-qa FYI I've commented out these tests temporarily as unknown apps won't have browser permissions displayed anymore, but this'll be restored in another PR.